File: programmers/js_src/level00/120850.js
// 문자열 정렬하기 (1)
function solution(my_string) {
  var answer = [];

  answer = my_string
    .replace(/[a-z]/g, '')
    .split('')
    .map(Number)
    .sort((a, b) => a - b);

  return answer;
}

File: programmers/java/level00/120850.java
// 문자열 정렬하기 (1)

import java.util.Arrays;

class Solution {
    public int[] solution(String my_string) {
        return Arrays.stream(my_string
                             .replaceAll("[^0-9]", "")
                             .split(""))
            .mapToInt(Integer::valueOf)
            .sorted()
            .toArray();
    }
}